build/docker/package.sh

```diff
@@ -4,6 +4,7 @@ set -euxo pipefail
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 VERSION=1.26.3
 BUILD_TYPE=${buildConfiguration:-Debug}
+ARCH=${ARCHITECTURE:-x64}
```

Comment:
Just an FYI, you can also set this default in the docker-compose file if it's preferable (not saying it is, just an FYI, as I've had to do similar in the Nuke branch) 🙂